ReeldIn/settings.py

```diff
@@ -16,15 +16,12 @@
 from dotenv import load_dotenv
 
 # Obtain the base directory of the project and use it to build the path to the .env file
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = Path(__file__).resolve().parent.parent
 dotenv_path = os.path.join(BASE_DIR, ".env")
 load_dotenv(dotenv_path)
 
 NPM_BIN_PATH = "C:/Program Files/nodejs/npm.cmd"
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
is the purpose of having two of these that
we set up the virtual environment and then set up the project itself...?

feel like this might be needed

Reply:
Oh no I think I just missed that a BASE_DIR field was already made when I was setting up the database and so I cleaned it up in this one